PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebCheckoutFunding.kt

```diff
@@ -0,0 +1,8 @@
+package com.paypal.android.checkoutweb
+
+/**
+ * Enum class to specify the type of funding for an order
+ */
+enum class PayPalWebCheckoutFunding(val value: String) {
+    CREDIT("credit"), PAY_LATER("paylater"), DEFAULT("")
```

Comment:
Let's add doc strings to each of these types to explain what each funding source means, and what the default is.